script/snow_share/health_check_procedure.sql

```diff
@@ -567,6 +572,7 @@ GRANT SELECT ON TABLE DATABASE_REPLICATION_USAGE_HISTORY to share S_SECURE_SHARE
 GRANT SELECT ON TABLE REPLICATION_GROUP_USAGE_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE SNOWPIPE_STREAMING_FILE_MIGRATION_HISTORY to share S_SECURE_SHARE;
 GRANT SELECT ON TABLE QUERY_HISTORY to share S_SECURE_SHARE;
+GRANT SELECT ON TABLE SESSIONS to share S_SECURE_SHARE_4825_2;
```

Comment:
This is wrong, why it is 4825_2, then share will not work

Reply:
@viswa-unravel that was a typo. corrected it now.